packages/apps/human-app/frontend/src/components/data-entry/multi-select.tsx

```diff
@@ -134,6 +134,11 @@ export function MultiSelect({
               onChange={(event) => {
                 handleChange(event as SelectChangeEvent<string[]>, field);
               }}
+              sx={{
+                '.MuiSvgIcon-root': {
```

Comment:
It works, but we shouldn't be passing internal element styles here.

If you check default theming on MUI - each component (including select) is able to change their colors (including things like this "arrow icon") when changing modes, hence if color theme is defined properly - you don't have to pass such "hacks".

Please attempt to fix it in the right place and use "hacks" only if it's last resort